_posts/2022-12-13-release.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Release 2022.50
+author: Priya
+date: 2022-12-13
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Show original column names.
+If a column is renamed, users may have difficulty finding its original name. To help with this, Mammoth now shows the original name in the column header when the user hovers over it.
```

Comment:
If a column is renamed in a View, it is difficult to find the original name. Mammoth now shows the original name in the column header on hover.

Reply:
fixed